std/experimental/allocator/building_blocks/allocator_list.d

```diff
@@ -632,119 +637,340 @@ version(Posix) @system unittest
     assert(b1.length == 1024 * 10);
 }
 
+/// Ditto
+shared struct SharedAllocatorList(Factory, BookkeepingAllocator = GCAllocator)
+{
+    import std.typecons : Ternary;
+    import std.traits : hasMember;
+    import core.internal.spinlock : SpinLock;
+
+private:
+    // Forward all calls to 'impl' and protect them by the lock below
+    AllocatorList!(Factory, BookkeepingAllocator) impl;
+    SpinLock lock = SpinLock(SpinLock.Contention.brief);
+
+    // Remove this when 'https://github.com/dlang/druntime/pull/2156' gets merged
+    static ref T assumeUnshared(T)(ref shared T val) @trusted @nogc pure nothrow
+    {
+        return *cast(T*) &val;
+    }
+
+    // Debug function used for testing
+    version(unittest)
+    auto allocators()
+    {
+        return impl.allocators;
+    }
+
+// Everything is inherited from the 'AllocatorList' implementation
+public:
+    /**
+    The alignment offered.
+    */
+    enum alignment = impl.alignment;
+
+    /**
+    Allocate a block of size `s`. First tries to allocate from the existing
+    list of already-created allocators. If neither can satisfy the request,
+    creates a new allocator by calling `make(s)` and delegates the request
+    to it. However, if the allocation fresh off a newly created allocator
+    fails, subsequent calls to `allocate` will not cause more calls to $(D
+    make).
```

Comment:
To be fair, nobody felt that need for the regular `AllocatorList` from which this comment was copied.

Reply:
After having a look at similar function in the package, I saw their common denominator was to not have their parameters documented. And this could have almost been on purpose – have a look at the package documentation of “building blocks” that provides a table documentating each of these functions in general.
<https://github.com/dlang/phobos/blob/b0f461be4f5a75ed2e988cfb95776390c661971e/std/experimental/allocator/building_blocks/package.d#L52>

Reply:
Apart from that, the text exactly describes the parameter meaning.